src/renderer/coremods/installer/commands.ts

```diff
@@ -0,0 +1,116 @@
+import { Injector, plugins } from "@replugged";
+import { ApplicationCommandOptionType } from "src/types";
+import { INSTALLER_SOURCES, InstallerSource, installFlow, installURL } from "./util";
+
+/**
+ * A map 'special' names that don't need to be translated by i18n
+ */
+const specialSourceDisplayNames: Partial<Record<InstallerSource, string>> = {
+  github: "GitHub",
+};
+
+function installSourceName(source: InstallerSource): string {
+  const displayName = specialSourceDisplayNames[source];
+  if (displayName) {
+    return displayName;
+  }
+
+  // TODO: i18n
+  return source;
+}
+
+export function loadCommands(injector: Injector): void {
+  injector.utils.registerSlashCommand({
+    name: "install",
+    // TODO: i18n
+    displayName: "install",
+    // TODO: i18n
+    description: "Install an addon",
+    options: [
+      {
+        name: "identifier",
+        // TODO: i18n
+        description: "The addon's updater id",
+        type: ApplicationCommandOptionType.String,
+        required: true,
+      },
+      {
+        name: "source",
+        // TODO: i18n
+        displayName: "source",
+        // TODO: i18n
+        description: `The addon source`,
+        type: ApplicationCommandOptionType.String,
+        required: false,
+        choices: INSTALLER_SOURCES.map((v) => ({
+          name: v,
+          displayName: installSourceName(v),
+          value: v,
+        })),
+      },
+      {
+        name: "id",
+        // TODO: i18n
+        displayName: "id",
+        // TODO: i18n
+        description: `A plugin ID that used by monorepo plugins only`,
+        type: ApplicationCommandOptionType.String,
+        required: false,
+      },
+    ],
+
+    async execute(args) {
+      await installFlow(
+        // @ts-expect-error identifier is a required argument, so no need for assertion
+        // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+        args.find((v) => v.name === "identifier").value as string,
+        args.find((v) => v.name === "source")?.value as InstallerSource | undefined,
+        args.find((v) => v.name === "id")?.value,
```

Comment:
Use getValue like we have in the commands file. It will take care of TS for you.

Reply:
args doesn't have getValue. This is in an `execute` function, not an `executor` (with execute you can just not give a response, I don't think thats possible with `executor`?)

Reply:
I'm pretty sure you can just return undefined?

Reply:
Not according to the types. Although, nullish values *are* handled. Should I include a void return type in this PR?

Reply:
just return an object with send value false I and nothing else would say